src/lib/hooks/useCurrency.ts

```diff
@@ -24,15 +24,15 @@ function parseStringOrBytes32(str: string | undefined, bytes32: string | undefin
     : defaultValue
 }
 
-export const UNKNOWN_TOKEN_SYMBOL = 'UNKNOWN'
+const UNKNOWN_TOKEN_SYMBOL = 'UNKNOWN'
 const UNKNOWN_TOKEN_NAME = 'Unknown Token'
```

Comment:
Now that these are both file-local, feels like they could just be inlined.  Especially since they are only used in one place.